### ISSUE TITLE
Random crashes when prompting the autocomplete

### ISSUE THREAD

**Issue by Weakky** [prismagraphql/graphql-playground#475]
#### This issue pertains to the following package(s):

- [x] GraphQL Playground - Electron App
- [ ] GraphQL Playground HTML
- [ ] GraphQL Playground
- [ ] GraphQL Playground Express Middleware
- [ ] GraphQL Playground Hapi Middleware
- [ ] GraphQL Playground Koa Middleware
- [ ] GraphQL Playground Lambda Middleware

#### What OS and OS version are you experiencing the issue(s) on?
macOS High Sierra 10.13.2

#### What version of graphql-playground(-electron/-middleware) are you experiencing the issue(s) on?
1.3.23

#### What is the actual behavior?

When prompting the autocomplete and writing some letters, the whole client randomly crashes leaving only the windows frame as if there we no components rendered.

#### What steps may we take to reproduce the behavior?

Unfortunately for now, I found no way to reproduce the issue reliably.

<img width="1312" alt="capture d ecran 2018-01-10 a 12 16 22" src="https://user-images.githubusercontent.com/25233379/34773696-eaea279e-f60c-11e7-82f4-b0d7dd521ec2.png">

**Comment by giautm** [comment]
Same issues here. :(

**Comment by marktani** [comment]
This just happened to me, with this output:

![image](https://user-images.githubusercontent.com/1780597/34837471-2fe57e86-f6fb-11e7-9c36-08552289bdd8.png)

**Comment by Weakky** [comment]
I am pretty sure this started happening since the latest version (for me at least), I never had these crashes before. Maybe looking at the latest commits could be a starting point?

**Comment by ymartin** [comment]
We're also experiencing this issue with the desktop version. Have there been any updates?

**Comment by liamdanielduffy** [comment]
@marktani is there a plan for fixing this? the app is difficult to be productive with because of the frequent crashes.

**Comment by mzuniga84** [comment]
I'm also having this issue. Any updates?

Thank you

**Comment by marktani** [comment]
I have not experienced this for quite some while anymore. Did anyone find a reproduction yet?

**Comment by dardub** [comment]
I just downloaded the latest Mac App and am experiencing it frequently. Reproducing is tough.

So I tried to reproduce by just quickly randomly pressing keys. Sometimes I could get it to crash and sometimes not. This last time, I was hitting a lot of random keys, I quit the playground before it crashed. Then I opened it up again and it crashes every time.

![graphql-playground-crash](https://user-images.githubusercontent.com/561983/37619543-0683a51c-2b77-11e8-8746-02883c9eb504.gif)

I put on verbose logging because I couldn't see any errors.

Does this help at all? Is there anything else I can provide?
